src/main/views/subscription-management.njk

```diff
@@ -38,8 +38,21 @@
             active: activeCourtTab
         }]
     }) }}
+
     {% if caseTableData | length == 0 and courtTableData | length == 0 %}
-        <p class="govuk-body">{{ noSubs }}</p>
+      {% if activeAllTab == true%}
+        <div class="message">
```

Comment:
any reason why are these wrapped in the div? Can we not give a paragraph message class since you are getting elements by that class name in the tests or use govuk-body class for instead?

Reply:
wrapper removed